src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -430,8 +424,8 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		let textOutput: string[] = [];
 		let elements = this.getHtmlElements();
 		elements.forEach(element => {
-			if (element && element.innerText) {
-				textOutput.push(element.innerText);
+			if (element && element.textContent) {
```

Comment:
can you describe the change from innerText to textContent?

Reply:
text content is what mark js is basing it's ranges for highlight. innerText sometimes is missing some spaces and newlines due to which the range is off. When we use textContent to calculate the ranges, we can directly use that range to highlight the correct range instead of marking all the elements with the keyword.